x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview.tsx

```diff
@@ -84,7 +79,6 @@ export const EngineOverview: React.FC = () => {
   }, [license, metaEnginesPage]);
 
   if (hasErrorConnecting) return <ErrorState />;
-  if (hasNoAccount) return <NoUserState />;
```

Comment:
One quick note about losing this UI state is we'll no longer be tracking users who encounter this state in our telemetry stats, which is a bit of a shame.

I'm curious as to whether we can access/increment telemetry as early as in the plugin setup - if so, it might be worth doing so there.

Reply:
Resolved via email thread - not worth doing as it might significantly adversely impact Kibana startup times.

Note to self: I should remove the `ui_error.no_as_account` telemetry field